modules/simple-payments/simple-payments.php

```diff
@@ -94,9 +95,11 @@ function output_shortcode( $data ) {
 <div class='{$data[ 'class' ]} jetpack-simple-payments-wrapper'>
 	<div class='jetpack-simple-payments-title'>{$data['title']}</div>
 	<div class='jetpack-simple-payments-description'>{$data['description']}</div>
```

Comment:
Can we have `<p>` inside of `div.jetpack-simple-payments-description`? Theme most likely has spacing for `<p>` and that would avoid to have an arbitrary margin for this specific shortcode.